### ISSUE TITLE
Magenta not working in Live 11 on Windows 11 for me

### ISSUE THREAD

**Issue by AfricanElectronic** [magenta/magenta#2036]
Anyone else having this issue with Magenta or know how to solve it? It won't show any or allow me to select any input clips/tracks.
![magenta](https://user-images.githubusercontent.com/122117448/211076303-8fb9cb76-055b-41f8-bce5-dda6d85cdba4.png)

**Comment by Camronius** [comment]
Yes same thing is happening to me. Used to work, within last few months it suddenly stopped.
